pconduit/v1/client/schema.go

```diff
@@ -23,20 +23,25 @@ import (
 	"github.com/conduitio/conduit-connector-protocol/pconduit/v1/toproto"
 	conduitv1 "github.com/conduitio/conduit-connector-protocol/proto/conduit/v1"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
 )
 
 type SchemaServiceClient struct {
 	grpcClient conduitv1.SchemaServiceClient
+	token      string
 }
 
 var _ pconduit.SchemaService = (*SchemaServiceClient)(nil)
 
-func NewSchemaServiceClient(cc *grpc.ClientConn) *SchemaServiceClient {
-	return &SchemaServiceClient{grpcClient: conduitv1.NewSchemaServiceClient(cc)}
+func NewSchemaServiceClient(cc *grpc.ClientConn, token string) *SchemaServiceClient {
+	return &SchemaServiceClient{
+		grpcClient: conduitv1.NewSchemaServiceClient(cc),
+		token:      token,
+	}
 }
 
 func (c *SchemaServiceClient) CreateSchema(ctx context.Context, request pconduit.CreateSchemaRequest) (pconduit.CreateSchemaResponse, error) {
-	ctx = internal.RepackConnectorTokenOutgoingContext(ctx)
```

Comment:
The usage of `RepackConnectorTokenOutgoingContext` (as was the case previously) assumes that the context parameter to this method contains the token, i.e. that the caller (connector/connector SDK) attaches it. However, it makes more sense that a client is configured with all the static configuration it needs and automatically attaches it.

Reply:
Won't we need this token in builtin plugins as well? How will we [handle name conflicts](https://github.com/ConduitIO/conduit/issues/1660) in the registry if we don't know which connector called the function?

Reply:
Maybe.:) But I'd leave that for when we actually come to that issue.

Reply:
Sorry but I strongly disagree. Let's figure this design out before we make changes that we'll have to revert later.